test/models/repaymentHistory/RepaymentHistoryUtilsSpec.scala

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.repaymentHistory
+
+
+import models.financialDetails.Payment
+import models.repaymentHistory.RepaymentHistoryUtils._
+import org.scalatest.Matchers
+import testUtils.{TestSupport}
+
+import java.time.LocalDate
+
+class RepaymentHistoryUtilsSpec extends TestSupport with Matchers {
+
+  val paymentsWithMFA: List[Payment] = List(
```

Comment:
really optional, but worth to consider moving data into ~Helper trait, in order to be able re-use data in the future etc;